cmd/metacache.go

```diff
@@ -69,6 +69,34 @@ type metacache struct {
 	dataVersion  uint8      `msg:"v"`
 }
 
+func (m metacache) matches(o listPathOptions, debugPrint func(msg string, data ...interface{})) bool {
```

Comment:
A pointer receiver is likely measurably faster since there are a bunch of value fields.